recipe/migrations/cuda102.yaml

```diff
@@ -0,0 +1,26 @@
+migrator_ts: 1576782497
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    0
+
+cuda_compiler_version:         # [linux64]
+  - None                       # [linux64]
+  - 9.2                        # [linux64]
```

Comment:
Do we want to keep this version? Are there any end of life for these versions?

Reply:
I think we want to keep this for now. Though agree we may wind up dropping it in the medium term.

Reply:
There aren't specific end of lifes for these versions. It is community driven as there are a lot of legacy or air-gap systems that still use 9.0/9.2 so depending on the users of a library they will continue to support those versions until the user base moves.

Reply:
CuPy still works with CUDA 8.0. This is necessary.

Reply:
I understand why CUDA 8.0 might be necessary to support Fermi. Why do you need CUDA 9.2?

Reply:
> It is community driven as there are a lot of legacy or air-gap systems that still use 9.0/9.2 

What do you mean? We are building the CUDA packages so that users don't have to.

Reply:
> I understand why CUDA 8.0 might be necessary to support Fermi. Why do you need CUDA 9.2?

This is the CUDA runtime version which newer versions will continue to run on older architectures. Older versions aren't necessary to support old architectures, just some codebases only work with or run best with certain versions of CUDA.